tests/conftest.py

```diff
@@ -285,6 +286,14 @@ def assert_collapsed(runtime: str) -> Callable[[Mapping[str, int], bool], None]:
     return partial(assert_function_in_collapsed, function_name)
 
 
+@fixture
+def assert_application_name(application_pid: int, runtime: str, in_container: bool) -> Generator:
+    desired_names = {"java": "java: /app/Fibonacci.jar", "python": "python: /app/lister.py"}
+    yield
+    if in_container and runtime in desired_names:
```

Comment:
Only `in_container`?

Reply:
Yep, I think that's enough, I refactored the applications running in docker images

Reply:
I don't understand this answer, can you elaborate?

Reply:
He means that he fixed the Dockerfiles of applications but not their execution when we run them not in a container. So he disables the check for non-containers

Reply:
Oh. It's pretty confusing, why not fix in the execution as well? Or at least add a `TODO` for clarification.

Reply:
Not worth the time. Can add a TODO

Reply:
added TODO